packages/ckeditor5-markdown-gfm/src/pastefrommarkdownexperimental.ts

```diff
@@ -7,11 +7,13 @@
  * @module markdown-gfm/pastefrommarkdownexperimental
  */
 
-import { type Editor, Plugin } from 'ckeditor5/src/core';
-import { type ClipboardInputTransformationEvent, ClipboardPipeline } from 'ckeditor5/src/clipboard';
+import { Plugin, type Editor } from 'ckeditor5/src/core';
+import { ClipboardPipeline, type ClipboardInputTransformationEvent } from 'ckeditor5/src/clipboard';
 import GFMDataProcessor from './gfmdataprocessor';
 import type { ViewDocumentKeyDownEvent } from 'ckeditor5/src/engine';
 
+const FORMATTING_TAGS = [ 'B', 'STRONG', 'I', 'EM', 'MARK', 'SMALL', 'DEL', 'INS', 'SUB', 'SUP' ];
```

Comment:
I'm not sure if this is enough. Maybe we could go in the opposite direction and try to filter out only wrappers that we know are just wrappers without meaning for the content, and then just check if there is no `<` in the filtered content. I'm afraid that the current approach could filter out too many tags. Is this safe for pasting multiple paragraphs or inline images inside the text?